### DISCUSSION TITLE
How to know the number of lines caption: will break the text into?

### DISCUSSION THREAD

**Discussion by srini8080** [ImageMagick/ImageMagick#6462] [Digital Image Processing]
Unix centos 8 stream
IM7

I have a long text that I am writing in a rectangle of fixed width and height. caption: is breaking the text automagically into multiple lines. How can I get the info on how many lines it broke it into and the actual text in each line?

Thanks

**Comment by snibgo** [comment]
For the number of lines, use `%[caption:lines]`, for example:
```
magick -size 20x200 caption:"A long line of text" -format %[caption:lines] info:
3
```
See https://imagemagick.org/script/escape.php

I don't think IM can tell you the text in each line. You could make an image, then use Tesseract on that image.

**Reply by srini8080** [reply]
Thanks much
